configure.ac

```diff
@@ -1914,6 +1914,19 @@ then
   fi
 fi
 
+# Look for parallel LTO support in GCC to speed up builds.
+AC_CACHE_CHECK([If GCC version supports parallel LTO],
```

Comment:
Would you like to check the possibility of using `AX_CHECK_COMPILE_FLAG` instead of the source build strategy?
`AX_CHECK_COMPILE_FLAG([-flto=auto]` could be good trial to check.

Please take a look similar usecase:
https://github.com/python/cpython/blob/d47584aae6fab1b767e2d2ea6611b9c0c3ff36e2/configure.ac#L1955

Reply:
Good one! I didn't know this existed.

Fix applied.